CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.7.0 (April 26, 2021)
+
+FEATURES:
+* Updated cloud-network/preview/2020-09-07 to include changes introduced in https://github.com/hashicorp/cloud-network/pull/503 (additional optional parameter for ListHVNRoutes - `destination`)
```

Comment:
Can you exclude the link to the network PR? It's private so not everyone will be able to access it

Reply:
Thanks for the reminder, done!